### PR TITLE
Pootle  pipeline

### PR DESCRIPTION
**Pull request by gradam**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -192,3 +195,12 @@ generate-server-po-files:
 
 load-server-po-files:
 	@docker exec -t sc-flask bash -c "cd server && python manage.py load_po_files"
+
+to-pootle:
+	@make generate-server-po-files
+	@docker exec -t sc-pootle-pipeline pipenv run python to_pootle.py
+	@docker exec -t sc-pootle bash -c "pootle update_stores"
+
+from_pootle:
```
Comment (TrebuhD) [inline Makefile]: 1. Should be dash, not underscore
2. Missing verb

File: Makefile [hunk 2]
```diff
@@ -192,3 +195,12 @@ generate-server-po-files:
 
 load-server-po-files:
 	@docker exec -t sc-flask bash -c "cd server && python manage.py load_po_files"
+
+to-pootle:
```
Comment (TrebuhD) [inline Makefile]: Missing variable (move(?)-to-pootle) / migrate / something like that

File: client/localization/localizer.py [hunk 3]
```diff
@@ -12,25 +13,33 @@ def to_pootle():
...
+*  from_pootle:
+    * you can pass extra argument: pootle_files path.
 
 example usage:
 python localizer.py to_pootle''')
 
 
 if __name__ == '__main__':
-    if len(sys.argv) != 2:
+    if 3 < len(sys.argv) < 1:
```
Comment (TrebuhD) [inline client/localization/localizer.py]: hmmmm

File: pootle/Dockerfile [hunk 4]
```diff
@@ -24,7 +25,9 @@ RUN /usr/local/bin/apt-install build-essential \
   curl \
   sudo \
   python-setuptools \
-  nano
+  vim \
+  nano \
```
Comment (TrebuhD) [inline pootle/Dockerfile]: both vim and nano are needed?

File: pootle/pootle.conf [hunk 5]
```diff
@@ -178,6 +178,8 @@ PARSE_POOL_SIZE = 200
 #                                }
 #                            })
 POOTLE_MARKUP_FILTER = ('html', {})
+POOTLE_ENABLE_API = True
```
Comment (TrebuhD) [inline pootle/pootle.conf]: I thought the API doesn't work?

File: pootle_pipeline/Dockerfile-pootle-pipeline [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+    ln -fs /opt/chromedriver-$CHROMEDRIVER_VERSION/chromedriver /usr/local/bin/chromedriver
+
+# Install google chrome
+RUN curl -sS -o - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - && \
+    echo "deb http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google-chrome.list && \
+    apt-get -yqq update && \
+    apt-get -yqq install google-chrome-stable && \
+    rm -rf /var/lib/apt/lists/*
+
+COPY . .
```
Comment (TrebuhD) [inline pootle_pipeline/Dockerfile-pootle-pipeline]: COPY . . ?

File: server/server/internationalization/generate_pootle.py [hunk 7]
```diff
@@ -21,24 +21,34 @@ def process_blurbs(db):
...
+        pot.append(polib.POEntry(
+            msgid=blurb['blurb'],
+            msgctxt=blurb['uid']
+        ))
 
-    (GENERATED_PO_FILES_DIR / 'blurbs').mkdir(parents=True, exist_ok=True)
-    po.save(str(GENERATED_PO_FILES_DIR / 'blurbs' / 'en.po'))
+    (GENERATED_PO_FILES_DIR / 'server_blurbs').mkdir(parents=True, exist_ok=True)
+    po.save(str(GENERATED_PO_FILES_DIR / 'server_blurbs' / 'en.po'))
+    # pot.save(str(GENERATED_PO_FILES_DIR / 'server_blurbs' / 'templates.po'))
```
Comment (TrebuhD) [inline server/server/internationalization/generate_pootle.py]: commented line

File: server/server/internationalization/generate_pootle.py [hunk 8]
```diff
@@ -89,16 +99,26 @@ def process_currencies(db):
...
+        pot.append(polib.POEntry(
+            msgid=name['name'],
+            msgctxt=name['symbol']
+        ))
 
-    (GENERATED_PO_FILES_DIR / 'currency_names').mkdir(parents=True, exist_ok=True)
-    po.save(str(GENERATED_PO_FILES_DIR / 'currency_names' / 'en.po'))
+    (GENERATED_PO_FILES_DIR / 'server_currency_names').mkdir(parents=True, exist_ok=True)
+    po.save(str(GENERATED_PO_FILES_DIR / 'server_currency_names' / 'en.po'))
+    # pot.save(str(GENERATED_PO_FILES_DIR / 'server_currency_names' / 'templates.pot'))
```
Comment (TrebuhD) [inline server/server/internationalization/generate_pootle.py]: ditto

File: server/server/internationalization/generate_pootle.py [hunk 9]
```diff
@@ -121,7 +141,7 @@ def run():
     Path(GENERATED_PO_FILES_DIR).mkdir(parents=True, exist_ok=True)
 
     process_blurbs(db)
-    process_menu(db)
+    # process_menu(db)
```
Comment (TrebuhD) [inline server/server/internationalization/generate_pootle.py]: ditto

File: client/localization/localizer.py (hunk 3 above)
Comment (gradam) [reply client/localization/localizer.py]: what do you mean by `hmmmmm`?

**Pull request merged**